underscore/underscore.d.ts

```diff
@@ -1017,6 +1002,11 @@ interface UnderscoreStatic {
 		predicate: _.ListIterator<T, boolean>,
 		context?: any): number;
 
+  findIndex<T, U extends {}>(
```

Comment:
`U extends {}` means the same thing as just `U`. it is rather confusing to use this for some declarations and not others. so i would recommend removing it.